actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -59,13 +59,11 @@ def initialize(routes)
         @routes = routes
       end
 
-      def format(formatter, filter = nil)
-        routes_to_display = filter_routes(filter)
-
+      def format(formatter, **options)
+        routes_to_display = filter_routes(options)
         routes = collect_routes(routes_to_display)
-
         if routes.none?
-          formatter.no_routes(collect_routes(@routes), filter)
+          formatter.no_routes(collect_routes(@routes), options[:filter])
```

Comment:
@vipulnsward agree with your suggestion to say `No routes found for this controller`, so we can kill the second argument here and don't have to capture the original filter.